exporter/coralogixexporter/config_test.go

```diff
@@ -315,3 +331,38 @@ func TestGetMetadataFromResource(t *testing.T) {
 	assert.Equal(t, "application", appName)
 	assert.Equal(t, "subsystem", subSystemName)
 }
+
+func TestCreateExportersWithBatcher(t *testing.T) {
+	factory := NewFactory()
+	cfg := factory.CreateDefaultConfig().(*Config)
+	cfg.Domain = "localhost"
+	cfg.PrivateKey = "test-key"
+	cfg.AppName = "test-app"
+	cfg.BatcherConfig.Enabled = true
+	cfg.BatcherConfig.FlushTimeout = 1 * time.Second
+	cfg.BatcherConfig.MinSizeItems = 100
+
+	// Test traces exporter
+	t.Run("traces_with_batcher", func(t *testing.T) {
+		set := exportertest.NewNopSettingsWithType(metadata.Type)
```

Comment:
this broke lint on main http://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/13461488652/job/37617739345

Reply:
@songy23 thanks for fixing it.
Since the tests passed and also in my machine, I didn't see the issue. Are we maybe skipping something from the tests in PRs?
Thanks again and sorry for the inconvenience.

Reply:
no worries, it's a merge conflict, things should be better once merge queue is enabled